modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -1036,7 +1038,10 @@ public ResultSet executeSqlQueryWithTimer(
 
         Exception failReason = null;
 
-        try (TraceSurroundings ignored = MTC.support(ctx.tracing().create(SQL_CMD_QRY_EXECUTE, MTC.span()))) {
+        try (TraceSurroundings ignored =
```

Comment:
It seems that it would be better to attach this tag to the root SQL_QRY span and set its value once during RunningQueryManager#register method execution. Something like - `run.span().addTag(SpanTags.SQL_QRY_ID, () -> Long.toString(qryId));`
WDYT?

Reply:
Good idea.